src/npctalk_funcs.cpp

```diff
@@ -346,6 +346,16 @@ void talk_function::assign_guard( npc &p )
     p.set_omt_destination();
 }
 
+void talk_function::abandon_camp( npc &p )
+{
+    ( void )p;
```

Comment:
```suggestion
```
Why even have this?

Reply:
Used to be required before I changed my mind on how to structure the abandon camp function.